### ISSUE TITLE
Battle movement text is different

### ISSUE THREAD

**Issue by ccmaster001** [sirgwain/craig-stars#89]
Ship movment is at least named different then is stars.
Stars 
0,5 - 2,5

![image](https://github.com/sirgwain/craig-stars/assets/170196074/aba4932f-1180-46a6-8d9e-c93aa247e71f)

**Comment by sirgwain** [comment]
Yeah, I should document it better. I always found the movement super confusing in Stars!. I implemented it the same, but it's basically like this:

Ship movement is stored as movement points divided over 4 round blocks, of which there are 4 of
those per battle (i.e. 16 rounds). A token's movement point is divided among the 4 round block, with movements being added in this order:

1, 3, 2, 4

A ship with 3 movement will move in the 1st, 3rd, and 2nd round, but not the fourth.

| movement/4 | 1   | 2   | 3   | 4   |
| ---------- | --- | --- | --- | --- |
| 2          | 1   | 0   | 1   | 0   |
| 3          | 1   | 1   | 0   | 1   |
| 4          | 1   | 1   | 1   | 1   |
| 5          | 2   | 1   | 1   | 1   |
| 6          | 2   | 1   | 2   | 1   |
| 7          | 2   | 2   | 1   | 2   |
| 8          | 2   | 2   | 2   | 2   |
| 9          | 3   | 2   | 2   | 2   |
| 10         | 3   | 2   | 3   | 2   |
